daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -319,11 +332,17 @@ enum PrimCon {
 // Builtin functions
 // Refer to DAML-LF major version 1 specification for types and behavior of those.
 enum BuiltinFunction {
-  ADD_DECIMAL = 0;
-  SUB_DECIMAL = 1;
-  MUL_DECIMAL = 2;
-  DIV_DECIMAL = 3;
-  ROUND_DECIMAL = 6;
+  ADD_DECIMAL = 0; // *Available in versions < 1.dev*
```

Comment:
Shouldn't this be deprecated rather than unavailable?

Reply:
fixed